src/components/PromisesTable.vue

```diff
@@ -79,14 +93,48 @@ export default {
     exclude: { type: Array, default: () => [] }
   },
   data: () => ({
-    search: ''
+    search: '',
+    filterStatus : '',
+    statusOptions: [
```

Comment:
I think we can host this value in an external file

Reply:
Just realized we have a file that already hosting those value